### PR TITLE
Fix links

### PR DESCRIPTION
**Pull request by tengqm**
This PR fixes some link errors as discovered by the `linkchecker.py` tool.

### CODE REVIEW

File: content/en/docs/concepts/architecture/mixed-version-proxy.md [hunk 1]
```diff
@@ -81,7 +82,9 @@ loads a special filter that does the following:
 ### How it works under the hood
 
 When an API Server receives a resource request, it first checks which API servers can
-serve the requested resource. This check happens using the internal `StorageVersion` API.
+serve the requested resource. This check happens using the internal
+[`StorageVersion` API](/docs/reference/generated/kubernetes-api/v{{ skew currentVersion
```
Comment (Shubham82) [inline content/en/docs/concepts/architecture/mixed-version-proxy.md:86]:
```suggestion
[`StorageVersion` API](/docs/reference/generated/kubernetes-api/v{{< skew currentVersion >
```

File: content/en/docs/tasks/configure-pod-container/static-pod.md [hunk 2]
```diff
@@ -71,34 +73,38 @@ For example, this is how to start a simple web server as a static Pod:
     ssh my-node1
     ```
 
-2. Choose a directory, say `/etc/kubernetes/manifests` and place a web server Pod definition there, for example `/etc/kubernetes/manifests/static-web.yaml`:
+2. Choose a directory, say `/etc/kubernetes/manifests` and place a web server
```
Comment (Shubham82) [inline content/en/docs/tasks/configure-pod-container/static-pod.md:76]:
```suggestion
1. Choose a directory, say `/etc/kubernetes/manifests` and place a web server
```

File: content/en/docs/tasks/configure-pod-container/static-pod.md [hunk 3]
```diff
@@ -71,34 +73,38 @@ For example, this is how to start a simple web server as a static Pod:
...
+         image: nginx
+         ports:
+           - name: web
+             containerPort: 80
+             protocol: TCP
+   EOF
+   ```
 
-3. Configure your kubelet on the node to use this directory by running it with `--pod-manifest-path=/etc/kubernetes/manifests/` argument. On Fedora edit `/etc/kubernetes/kubelet` to include this line:
+3. Configure your kubelet on the node to use this directory by running it with
```
Comment (Shubham82) [inline content/en/docs/tasks/configure-pod-container/static-pod.md:100]:
```suggestion
1. Configure your kubelet on the node to use this directory by running it with
```

File: content/en/docs/tasks/configure-pod-container/static-pod.md [hunk 4]
```diff
@@ -138,18 +144,20 @@ To use this approach:
               protocol: TCP
     ```
 
-2. Configure the kubelet on your selected node to use this web manifest by running it with `--manifest-url=<manifest-url>`. On Fedora, edit `/etc/kubernetes/kubelet` to include this line:
+2. Configure the kubelet on your selected node to use this web manifest by
```
Comment (Shubham82) [inline content/en/docs/tasks/configure-pod-container/static-pod.md:147]:
```suggestion
1. Configure the kubelet on your selected node to use this web manifest by
```

File: content/en/docs/tasks/configure-pod-container/static-pod.md [hunk 5]
```diff
@@ -138,18 +144,20 @@ To use this approach:
...
+   On Fedora, edit `/etc/kubernetes/kubelet` to include this line:
 
-    ```
-    KUBELET_ARGS="--cluster-dns=10.254.0.10 --cluster-domain=kube.local --manifest-url=<manifest-url>"
-    ```
+   ```shell
+   KUBELET_ARGS="--cluster-dns=10.254.0.10 --cluster-domain=kube.local --manifest-url=<manifest-url>"
+   ```
 
 3. Restart the kubelet. On Fedora, you would run:
```
Comment (Shubham82) [inline content/en/docs/tasks/configure-pod-container/static-pod.md:155]:
```suggestion
1. Restart the kubelet. On Fedora, you would run:
```

**Review by Shubham82 — CHANGES_REQUESTED** [review]

File: content/en/docs/concepts/architecture/mixed-version-proxy.md (hunk 1 above)
Comment (Shubham82) [reply content/en/docs/concepts/architecture/mixed-version-proxy.md:86]: @tengqm small nit here.

File: content/en/docs/tasks/configure-pod-container/static-pod.md (hunk 2 above)
Comment (Shubham82) [reply content/en/docs/tasks/configure-pod-container/static-pod.md:76]: here we can change the number to 1.

File: content/en/docs/tasks/configure-pod-container/static-pod.md (hunk 3 above)
Comment (Shubham82) [reply content/en/docs/tasks/configure-pod-container/static-pod.md:100]: we also have to change 4 no to 1 at line number 111.

File: content/en/docs/tasks/configure-pod-container/static-pod.md (hunk 4 above)
Comment (Shubham82) [reply content/en/docs/tasks/configure-pod-container/static-pod.md:147]: @tengqm you missed this one

File: content/en/docs/tasks/configure-pod-container/static-pod.md (hunk 5 above)
Comment (Shubham82) [reply content/en/docs/tasks/configure-pod-container/static-pod.md:155]: also missed this one.

File: content/en/docs/tasks/configure-pod-container/static-pod.md (hunk 4 above)
Comment (tengqm) [reply content/en/docs/tasks/configure-pod-container/static-pod.md]: To be honest, this numbering issue is not relevant to the current PR.
I'd more inclined to have a PR self-contained and focused.
With that, I'm happy to fix the numbering issue in a new commit.

**Comment by Shubham82** [comment]
Thanks @tengqm 
/lgtm

**Pull request merged**